16/umbraco-cms/customizing/extending-overview/extension-types/kind.md

```diff
@@ -115,9 +122,8 @@ const manifest = {
 	},
 };
 
-extensionRegistry.register(manifest);
+umbExtensionsRegistry.register(manifest);
```

Comment:
Again, I think we should stop putting the registration code in the examples, just keep the manifest